lib/Jsrt/JsrtDiag.cpp

```diff
@@ -671,4 +672,19 @@ CHAKRA_API JsDiagEvaluate(
 
     }, false /*allowInObjectBeforeCollectCallback*/, true /*scriptExceptionAllowed*/);
```

Comment:
@agarwal-sandeep thanks for the details. In that case, I will apply these changes and enable that for xplat too.